en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/11-upgrading-to-elasticsearch-73.markdown

```diff
@@ -108,3 +104,89 @@ procedure takes only a few steps:
+
+    ```yaml
+    path.repo: [ "/path/to/elasticsearch_local_backup" ]
+    ```
+
+3. Restart your Elasticsearch nodes
+
+4. [Register the snapshot repository](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-register-repository.html): run the following API request (for example through the Dev Tools console in Kibana):
+
+    ```
+    PUT /_snapshot/elasticsearch_local_backup
+    {
+      "type": "fs",
+      "settings": {
+        "location": "/path/to/elasticsearch_local_backup"
+      }
+    }
+    ```
+
+5. [Create a snapshot](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-take-snapshot.html):
+
+    ```
+    PUT /_snapshot/elasticsearch_local_backup/snapshot1?wait_for_completion=true
+    {
+      "indices": "liferay-search-tuning*",
+      "ignore_unavailable": true,
+      "include_global_state": false
+    }
+    ```
+
+    If you want to create a snapshot for all Liferay indexes, you can use `"indices": "liferay*,workflow-metrics*"` instead.
+
+6. To [restore](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-restore-snapshot.html) only specific indexes from a snapshot under a different name, run an API call similar to this:
+
+    ```
+    POST /_snapshot/elasticsearch_local_backup/snapshot1/_restore
+    {
+      "indices": "liferay-20101-search-tuning-synonyms,liferay-20101-search-tuning-rankings",
+      "ignore_unavailable": true,
+      "include_global_state": false,
+      "rename_pattern": "(.+)",
+      "rename_replacement": "restored_$1",
+      "include_aliases": false
+    }
+    ```
+
+    where `indicies` sets the index names to restore from (as they are stored in the snapshot!). The indexes from the above call will be restored as `restored_liferay-20101-search-tuning-rankings` and `liferay-20101-search-tuning-synonyms` according to the `rename_pattern` and `rename_replacement` regular expressions.
+
+Restoring the _Search Tuning_ indexes from a snapshot under different names comes in handy, if you did not upgrade your Elasticsearch 6 cluster, but rather, you just set-up your new Elasticsearch 7 cluster, configured Elasticsearch 7 in @product-ver@ and performed a full reindex from @product@.
+
+In this case, existing _Search Tuning_ entries will be missing, because whose indexes will be empty.
+
+To restore your existing _Search Tuning_ index documents, you can use the [Reindex API](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html#docs-reindex) of Elasticsearch, like this:
+
+```
+   POST _reindex/
+   {
+     "dest": {
+       "index": "liferay-20101-search-tuning-synonyms"
+     },
+     "source": {
+       "index": "restored_liferay-20101-search-tuning-synonyms"
+     }
+   }
+```
+
+Note: it is possible to create and manage snapshots through Kibana's UI in [Kibana 7.x](https://www.elastic.co/guide/en/kibana/7.x/snapshot-repositories.html).
+
+### Default Search Tuning Elasticsearch Index Names
+
```

Comment:
Do you have a Grow article with this information?
I created a table out of this but I'm not sure it's accurate (what was the synonyms name on 7.2 SP1?):

![tuning-index-names](https://user-images.githubusercontent.com/1728666/98609667-cc7e7380-22bb-11eb-82d0-2dc15e0fd307.png)

Reply:
See https://help.liferay.com/hc/en-us/articles/360048464852#questions--troubleshooting

Reply:
Yes! I knew we had this somewhere. Thanks.